hello.js

```diff
@@ -0,0 +1,6 @@
+let x = Myfunction(4, 3);
```

Comment:
The function name should be in camelcase. Please change it.

Reply:
changes are done.please check